modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -85,6 +85,16 @@
 
     private final List<BucketDefinition> buckets = new ArrayList<>();
 
+    private Integer dataMemoryQuota;
+
+    private Integer indexMemoryQuota;
+
+    private Integer searchMemoryQuota;
+
+    private Integer analyticsMemoryQuota;
```

Comment:
I think for now it might be better if we do not include analytics or eventing in this change, since the other parts of the code do not have support for it (i.e. you cannot enable those services on the other apis). I think we should do follow-up changes to add support for analytics and/or eventing if we need to.

Reply:
makes sense